bokeh/embed/bundle.py

```diff
@@ -420,7 +420,14 @@ def _use_mathjax(objs: Sequence[Model | Document]) -> bool:
         bool
     '''
     from ..models.text import MathText
-    return _any(objs, lambda obj: isinstance(obj, MathText))
+
+    def model_require_mathjax(model: Model) -> bool:
+        ''' If any model explicit require MathJax with enable_tex option set to true
+        '''
+        properties = model._property_values
+        return properties.get('enable_tex', False)
```

Comment:
Instead of collecting all properties and their values, you should ask if a property exists and if so, just access it as a regular attribute, e.g.:
```py
return "enable_tex" in model.properties() and model.enable_tex
```